src/backend/libpq/rangerrest.c

```diff
@@ -468,7 +440,7 @@ void call_ranger_rest(CURL_HANDLE curl_handle, char* request)
     }
     else
     {
-        elog(LOG, "%lu bytes retrieved from Ranger Restful API.",
+        elog(LOG, "%d bytes retrieved from Ranger Restful API.",
```

Comment:
It should be, but this pull request is just a hotfix and small refactor. I think ranger codes should be largely refactored later on which is not relevant to this pull request. Thanks.